poc_iot_verifier/src/loader.rs

```diff
@@ -64,7 +74,7 @@ impl Loader {
         &mut self,
         shutdown: &triggered::Listener,
         gateway_cache: &GatewayCache,
-    ) -> Result {
+    ) -> anyhow::Result<()> {
```

Comment:
why not import anyhow result and avoid the need for all the explicit type annotations here and in other modules ?

Reply:
Since the `std::result::Result` is the default when we’re not importing our own thiserror module based result type and we’re using that one more often throughout the app’s other modules, I appreciate the explicit call out